.travis.yml

```diff
@@ -107,6 +107,19 @@ jobs:
       # gcc on Travis claims to not be CMAKE_COMPILER_IS_GNUCC so we only run clang.
       compiler: clang
       env: DYNAMORIO_CROSS_AARCHXX_LINUX_ONLY=no DEPLOY=no EXTRA_ARGS=64_only
+      after_script:
+        # Travis logs for OSX builds are truncated because the VM exits early.
+        # Sleep to allow Travis to capture complete log. More context on
+        # DynamoRIO/dynamorio#4223.
```

Comment:
nit: Our convention (from before the move to github) has been `i#4223`; ditto below.  That is what I search for in the code.

Reply:
Fixed it. Thanks.